deno.jsonc

```diff
@@ -31,5 +30,10 @@
             "./package-lock.json",
             "./bundling/bundles"
         ]
+    },
+    "imports": {
+        "@std/path": "jsr:@std/path@^0.221.0",
+        "grammy_types": "https://deno.land/x/grammy_types@v3.6.2/mod.ts",
+        "debug": "https://cdn.skypack.dev/debug@4.3.4"
```

Comment:
grammY is a library. You cannot use import maps with libraries. Please revert.

You can technically use them with JSR, but this is a feature of JSR that can rewrite parts of the library as a build step that is based on import maps. The actual library that people later import from JSR does not contain an import map anymore.

Reply:
Maybe like this: <https://github.com/oakserver/oak/blob/c8e535fdec9bb8155cdbf92c2ad8172ac1b9e893/examples/closeServer.ts#L6>

Reply:
That does not work on /x yet.